FinalCalculator

```diff
@@ -0,0 +1,63 @@
+import math
+ 
+#by Aditya Kumar and Sravan Tallapaka
```

Comment:
# Great work!
> Score: 9/10 (+ 7 for completion of the task and +2 for extensive functions and a user-friendly experience)
## Potential Improvements:
- Work to get Order of Operations natively ie. code it or use parentheses
- Add a GUI ;)
 ~ Ojasw Upadhyay